test/controllers/csvs_controller_test.rb

```diff
@@ -0,0 +1,335 @@
+require_relative '../test_helper'
+
+describe CsvsController do
+  let(:deployer) { users(:deployer) }
+  let(:export) { csv_exports(:pending) }
+
+  describe "permissions test" do
+    as_a_admin do
+      describe "a GET to new" do
+        it "renders the page with user link & admin menu" do
+          get :new
+          assert_template :new
+          assert @response.body.include? "Create Deploys CSV Report"
+          assert @response.body.include? "Download Users CSV Report"
+          assert @response.body.include? "Admin"
+          assert @response.body.include? "Environment variables"
+        end
+      end
+    end
+
+    as_a_deployer do
+      describe "a GET to new" do
+        it "renders the page without user link & limited admin menu" do
+          get :new
+          assert_template :new
+          assert @response.body.include? "Create Deploys CSV Report"
```

Comment:
extract the labels to `let` blocks so they do not drift / only 1 gets changed when a test starts breaking